packages/flutter_tools/templates/create/.idea/libraries/Dart_SDK.xml.tmpl

```diff
@@ -0,0 +1,9 @@
+<component name="libraryTable">
+  <library name="Dart SDK">
+    <CLASSES>
+      <root url="{{flutterRootUri}}/bin/cache/dart-sdk/lib/core" />
```

Comment:
Including only core folder is not enough. The full list of SDK roots on Windows looks like pasted below. Note that there are 2 slashes after `file:`. On Mac/Linux I guess there will be 3 slashes after `file:` because absolute paths start with slash.
```
<component name="libraryTable">
  <component name="libraryTable">
  <library name="Dart SDK">
    <CLASSES>
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/async" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/collection" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/convert" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/core" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/developer" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/html" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/indexed_db" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/io" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/isolate" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/js" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/js_util" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/math" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/mirrors" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/svg" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/typed_data" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/web_audio" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/web_gl" />
      <root url="file://C:/work/flutter/bin/cache/dart-sdk/lib/web_sql" />
    </CLASSES>
    <JAVADOC />
    <SOURCES />
  </library>
</component>
```